src/constraints.jl

```diff
@@ -94,7 +94,7 @@ function _dual_start(
             <:MOI.AbstractScalarSet,
         },
     },
-)::Union{Nothing,Float64}
+)::Union{Nothing,Number}
```

Comment:
So this is an interesting one. What is the dual start of a `ScalarAffineFunction{Complex{Float64}}-in-EqualTo{Complex{Float64}` constraint?